README.md

```diff
@@ -36,19 +36,18 @@ Bergamot translator provides a unified API for ([Marian NMT](https://marian-nmt.
     cd bergamot-translator
     ```
 
-3. Download files (only required if you want to package files in wasm binary)
+3. Download files (only required if you want to perform inference using build artifacts)
```

Comment:
Why else would anyone want to use the build artifacts? :) Also, I think it is a bit inaccurate, since inference is possible without the vocabs, it is just slower. Maybe something like "to improve the translation speed"?

Reply:
> Why else would anyone want to use the build artifacts? :) 

I agree with you on that one. I will remove it.

> Also, I think it is a bit inaccurate, since inference is possible without the vocabs, it is just slower. Maybe something like "to improve the translation speed"?

@motin Inference is not possible without vocabs. So that statement is correct 👍

Reply:
I think you're confusing vocabs and lexical shortlists.  Vocabs are required.  Lexical shortlists are optional.

Reply:
Ah yes indeed I am! Thanks @abhi-agg and @kpu for clarifying!